flowmachine/flowmachine/core/server/query_schemas/flows.py

```diff
@@ -58,3 +81,21 @@ class FlowsSchema(BaseSchema):
     join_type = fields.String(validate=OneOf(Join.join_kinds), missing="inner")
 
     __model__ = FlowsExposed
+
+
+class InflowsSchema(BaseSchema):
+    query_kind = fields.String(validate=OneOf(["flows"]))
```

Comment:
Can just subclass `FlowsSchema` and override `query_kind` to fields.String(validate=OneOf(["inflows"])) and `__model__ = FlowsExposed`?

Reply:
Done - also applied similar structure to InflowsExposed and Outflows Exposed